src/cargo/core/workspace.rs

```diff
@@ -68,7 +68,10 @@ enum MaybePackage {
 pub enum WorkspaceConfig {
     /// Indicates that `[workspace]` was present and the members were
     /// optionally specified as well.
-    Root { members: Option<Vec<String>> },
+    Root {
+        members: Option<Vec<String>>,
+        exclude: Vec<String>,
```

Comment:
We could yeah, but we don't have checks like that for `members` and other such keys. I could imagine that in debugging situations it's useful to have absolute paths (although certainly never committed)